.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Remember, an issue is not the place to ask questions. You can use [Stack Overflow](http://stackoverflow.com/questions/tagged/cakephp) for that.
+
+Before you open an issue, please check if a similar issue already exists or has been closed before.
+
+### When reporting a bug, please be sure to include the following:
+- [ ] A descriptive title
+- [ ] What version of CakePHP you're using, and the platform(s) you're running it on
+- [ ] The behavior you expect to see, and the actual behavior
+- [ ] An *isolated* way to reproduce the behavior
+
+### When you open an issue for a feature request, please add as much detail as possible:
+- [ ] A descriptive title
+- [ ] A description of the problem you're trying to solve, including *why* you think this is a problem
+- [ ] An overview of the suggested solution
+- [ ] If the feature changes current behavior, reasons why your solution is better
```

Comment:
I agree, should be simple template things, not instructions. Instructions should be kept in the Contribution.md file etc.